### PR TITLE
Fix wording related to adapters supporting database comments [ci skip]

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
r? @jeremy

### CODE REVIEW

File: guides/source/active_record_migrations.md [hunk 1]
```diff
@@ -246,7 +246,7 @@ class AddUserRefToProducts < ActiveRecord::Migration[5.0]
 end
 ```
 
-This migration will create a `user_id` column and appropriate index.   
+This migration will create a `user_id` column and appropriate index.
```
Comment (vipulnsward) [inline guides/source/active_record_migrations.md]: @prathamesh-sonpatki some editor setting causing these?

File: guides/source/active_record_migrations.md [hunk 2]
```diff
@@ -361,7 +361,7 @@ that will be stored in database itself and can be viewed with database administr
 tools, such as MySQL Workbench or PgAdmin III. It's highly recommended to specify
 comments in migrations for applications with large databases as it helps people
 to understand data model and generate documentation.
-Currently only MySQL and PostgreSQL supports comments.
+Currently only MySQL and PostgreSQL adapters support comments.
```
Comment (jeremy) [inline guides/source/active_record_migrations.md]: Currently only the* MySQL and PostgreSQL adapters support comments.

Comment (prathamesh-sonpatki) [inline guides/source/active_record_migrations.md]: Thanks updated.

File: guides/source/active_record_migrations.md (hunk 1 above)
Comment (prathamesh-sonpatki) [inline guides/source/active_record_migrations.md]: Yes. My editor is configured to remove extra whitespace. Good to remove no?

Comment (vipulnsward) [inline guides/source/active_record_migrations.md]: Seeing these in many past changes. This skews commit history, and hard to debug issues.

**Pull request merged**